proxy-node-gateway/src/main/java/uk/gov/ida/notification/exceptions/mappers/ExceptionToErrorPageMapper.java

```diff
@@ -1,27 +1,23 @@
 package uk.gov.ida.notification.exceptions.mappers;
 
-import org.joda.time.DateTime;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import uk.gov.ida.notification.shared.ProxyNodeLogger;
 
 import javax.ws.rs.core.Context;
 import javax.ws.rs.core.Response;
 import javax.ws.rs.core.UriInfo;
 import javax.ws.rs.ext.ExceptionMapper;
 import java.net.URI;
-import java.util.Optional;
-import java.util.concurrent.ThreadLocalRandom;
+import java.util.logging.Level;
 
 import static java.text.MessageFormat.format;
 
 public abstract class ExceptionToErrorPageMapper<TException extends Exception> implements ExceptionMapper<TException> {
 
-    private static final Logger LOG = LoggerFactory.getLogger(ExceptionToErrorPageMapper.class);
+    private final ProxyNodeLogger proxyNodeLogger = new ProxyNodeLogger();
```

Comment:
I'd say we should pass in an instance of `ProxyNodeLogger` in `GatewayApplication` but this is fine if we're going to try to make `ProxyNodeLogger` static in a future PR